packages/data-models/src/language.js

```diff
@@ -9,7 +9,8 @@ export const Lang = {
   PERSIAN: 'per',
   GEEZ: 'gez',
   CHINESE: 'zho',
-  SYRIAC: 'syr'
+  SYRIAC: 'syr',
+  ENGLISH: 'eng'
```

Comment:
I think we still have more cleanup to do here with the language codes of the alpheios supported languages --- right now this list has to be updated in too many places when we add support for a new source language. (see https://github.com/alpheios-project/documentation/blob/master/development/adding_a_language.md which is almost certainly out of date). I'm not sure what the best solution is and I don't think it has to be solved with this PR but we should keep sight of it.

Reply:
We could probably move the discussion to the https://github.com/alpheios-project/alpheios-core/issues/597 and handle this issue separately because it might require some analysis in order to find the best solution.

Reply:
ok